bootstrap5-demo-with-translate/README.md

```diff
@@ -0,0 +1,15 @@
+## Bootstrap 4 
+Common Bootstrap 4 demo using `ngx-translate` that allows for using multiple languages (locales) and switch them on the fly.
+
+### Install and Start Demo
+```bash
+git clone https://github.com/ghiscoding/angular-slickgrid-demos
+cd bootstrap4-demo
```

Comment:
also make sure the folder name is correct, I assume it should be `cd bootstrap5-demo-with-translate` (which makes me think the BS4 folder name is incorrect)

Reply:
Fixed directory paths for all Bootstrap demos in https://github.com/ghiscoding/angular-slickgrid-demos/pull/5/commits/14181c39cef77434ce50556d6dca53ab9feee981, https://github.com/ghiscoding/angular-slickgrid-demos/pull/5/commits/ee92c60c6348191271c7a3589d0dbb7e2e7ce2e6, and https://github.com/ghiscoding/angular-slickgrid-demos/pull/5/commits/dd7f93d132c4ccbe75e2103998b57c69e548725f